starter_code/person.js

```diff
@@ -1,5 +1,8 @@
 class Person {
   constructor(name, originFloor, destinationFloor){
+    this.name = name;
+    this.originFloor = originFloor;
+    this.destinationFloor = destinationFloor;
   }
 }
 
```

Comment:
#Discussed by Thor and solution is send yesterday. If you have additional questions let us know.